test/helpers/error.go

```diff
@@ -28,10 +28,17 @@ func CombineErrors(errs []error) error {
 	if len(errs) == 0 {
 		return nil
 	}
+	isErr := false
 	var sb strings.Builder
 	for _, err := range errs {
-		sb.WriteString(err.Error())
-		sb.WriteString("\n")
+		if err != nil {
+			isErr = true
+			sb.WriteString(err.Error())
+			sb.WriteString("\n")
+		}
+	}
+	if !isErr {
```

Comment:
Can't you just check `sb.Len()` instead of having a boolean flag?

Reply:
Changed to using an array to save the error messages, and check `if len(msgs) == 0`